Sometimes after ts_open/ts_close/ts_open on the first
sample(s) dejitter will crash (sorry no backtrace)

memsetting tslib_dejitter so djt->nr is not random anymore


#
# Patch managed by http://www.holgerschurig.de/patcher.html
#

--- tslib/plugins/dejitter.c~initialize_djs
+++ tslib/plugins/dejitter.c
@@ -205,6 +205,7 @@
 	if (djt == NULL)
 		return NULL;
 
+	memset(djt, 0, sizeof(struct tslib_dejitter));
 	djt->module.ops = &dejitter_ops;
 
 	djt->delta = 100;
